OpenRA.Mods.Common/Traits/Air/AttackAircraft.cs

```diff
@@ -15,9 +15,15 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	// TODO: Add CurlyShuffle (TD, TS), Circle (Generals Gunship-style)
+	public enum AirAttackType { Hover, FlyBy }
+
 	public class AttackAircraftInfo : AttackFollowInfo, Requires<AircraftInfo>
 	{
-		[Desc("Delay, in game ticks, before non-hovering aircraft turns to attack.")]
+		[Desc("Attack behavior. Currently supported types are FlyBy (default) and Hover.")]
+		public readonly AirAttackType AttackType = AirAttackType.FlyBy;
```

Comment:
I really like this idea for defining different attack patterns here. I think I will try to generalize this when I get around to the Attack* refactor. If we're going to nitpick names anyway, I think the proper terminology for such a maneuver would be `Strafe` instead of `FlyBy`.

Reply:
Done.